arrows/core/metadata_map_io_csv.cxx

```diff
@@ -24,6 +31,49 @@ namespace arrows {
 
 namespace core {
 
+namespace {
+
+// ----------------------------------------------------------------------------
+template < typename out >
+void
+split( std::string const& s, char delim, out result )
+{
+  std::istringstream iss( s );
+  std::string item;
+
+  while( std::getline( iss, item, delim ) )
+  {
+    *result++ = item;
+  }
+}
+
+// ----------------------------------------------------------------------------
+std::vector< std::string >
+split( std::string const& s, char delim )
```

Comment:
It's annoying that this is not a C++ standard function.  We shouldn't have to reimplement.  We do have utils for this:
https://github.com/Kitware/kwiver/blob/master/vital/util/tokenize.h#L42